src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -386,21 +383,21 @@
   "com.minecolonies.command.citizeninfo.workposnull": "§2Work position: §4No work position found!",
   "com.minecolonies.command.citizeninfo.pos": "§2Citizen position: §4x=§f%s §4y=§f%s §4z=§f%s",
   "com.minecolonies.command.citizeninfo.notloaded": "Citizen entity not loaded!",
-  "com.minecolonies.command.citizeninfo.notfound": "Wrong citizen id, no citizen found.",
-  "com.minecolonies.command.citizenkill.success": "Has been removed.",
+  "com.minecolonies.command.citizeninfo.notfound": "Wrong citizen ID, no citizen found.",
+  "com.minecolonies.command.citizenkill.success": "Citizen %s has been killed.",
```

Comment:
Another case where I believe the name is prepended.  I'll check.

Reply:
Ok, this appears to be the response that happens on a console command, so it's confirmation of a very targeted action. Probably not worth changing.